tests/distributed/test_pynccl.py

```diff
@@ -151,6 +152,68 @@ def test_pynccl_with_cudagraph():
     distributed_run(worker_fn_with_cudagraph, 2)
 
 
+@worker_fn_wrapper
+def pp_worker_fn():
```

Comment:
name it as `send_recv` worker? it is not testing pp though.

Reply:
Ok will change - this is following convention from here:

https://github.com/andoorve/vllm/blob/8824d36656e52507f341904df23c9475aaf8c0e7/tests/distributed/test_pynccl.py#L72

Reply:
you can also change that into allreduce worker.

Reply:
Done